frontend/include/chpl/framework/ErrorWriter.h

```diff
@@ -393,6 +393,9 @@ class ErrorWriter : public ErrorWriterBase {
   std::ostream& oss_;
   ErrorWriterBase::OutputFormat outputFormat_;
   bool useColor_;
+  std::string lastFilePath_;
+
+  bool noteLastFilePath(std::string newPath);
```

Comment:
Based on how this is used, I recommend calling it `noteFilePath`.  A comment about the purpose of the return value would be great.